CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
+* Starting Kubernetes 1.29, the default cgroups implementation on Azure Linux AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance, and read the [FAQ](https://learn.microsoft.com/troubleshoot/azure/azure-kubernetes/aks-increased-memory-usage-cgroup-v2) for cgroupsv2.
+* Staring with the `2024-01-01` and `2024-01-02-preview` APIs, we will begin to reject unknown fields in the request payloads.
+* Changes to reduce the kube-reserved memory reservation and eviction threshold will not be available in 1.28 as previously shared due to a release issue. These optimizations will be releasing with AKS Kubernetes minor version 1.29, which previews in January 2024.
+ 
+### Release notes
+
+* Bug Fixes 
+  * Fix for artifact streaming update related panic when nil and switches testing to use params/test package functions.
+  * Fix for LTS to look at AgentPoolResources.
+  * Fix for a put MC operation with an API version that doesn't have the servicemeshprofile triggers an "invalid mode" error during RP validations.
+
+* Behavioral Change
+  * ManagedCluster, AgentPool are now deprecated fields in agentpool goals.
+  * The default ResponseHeaderTimeout is now set to 55s to prevent abnormal slow operation caused by hanging connection.
+  * Community images are now added into reduction pipelines for cost saving.
+  * The memory limit for [Azure Key Vault provider for Secrets Store CSI Driver](https://learn.microsoft.com/en-us/azure/aks/csi-secrets-store-driver) is now increased to 300Mi.
+  * Expanders flag is removed from AutoscalerProfile.
+  * Config checksum is added to cns daemonset spec.
+  * Orchestrator version is moved to new validator removing from server/validation.
+
+* Component Updates
+  * Windows Kubernetes CVE fixes for 3P Windows customers, for: [CVE-2023-5528](https://github.com/advisories/GHSA-hq6q-c2x6-hmch)
+  * Update ama-logs addon image to 3.1.16
+  * Bumpe cloud-controller-manager image to [v1.25.23](https://github.com/kubernetes-sigs/cloud-provider-azure/releases/tag/v1.25.23), [v1.26.18](https://github.com/kubernetes-sigs/cloud-provider-azure/releases/tag/v1.26.18), [v1.27.12](https://github.com/kubernetes-sigs/cloud-provider-azure/releases/tag/v1.27.12), [v1.28.4](https://github.com/kubernetes-sigs/cloud-provider-azure/releases/tag/v1.28.4).
+  * Bump agentbaker version to v0.20231122.3, v0.20231206.0.
```

Comment:
We can remove this line and we just need to cover the vhd notes

Reply:
Updated